crates/conformance-test-utils/src/lib.rs

```diff
@@ -1 +1,31 @@
 pub mod kakarot;
+pub mod madara;
+
```

Comment:
Might make more sense to have a separate file `types.rs`.

Reply:
gotcha, that's better let me move it to a separate file 👍